usaspending_api/etl/subaward_etl.py

```diff
@@ -0,0 +1,246 @@
+import logging
```

Comment:
Thank you for splitting this out into a new file; load_submission.py is so big already.